2023-08-16 18:02:25.635402 <class 'spotipy.exceptions.SpotifyException'>
Traceback (most recent call last):
  File "D:\Programming\Python\Music Downloader\music_fetcher.py", line 21, in get_spotify_playlist_songs
    playlist = spotify.playlist(playlist_id=playlist_url, fields="tracks(items), name")
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Programming\Python\Music Downloader\venv\Lib\site-packages\spotipy\client.py", line 650, in playlist
    plid = self._get_id("playlist", playlist_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Programming\Python\Music Downloader\venv\Lib\site-packages\spotipy\client.py", line 1981, in _get_id
    raise SpotifyException(400, -1, "Unexpected Spotify URL type.")
spotipy.exceptions.SpotifyException: http status: 400, code:-1 - Unexpected Spotify URL type., reason: None


-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

